scripts/test_screeps_rl_simulator_harness.py

```
@@ -1814,6 +1814,73 @@ def test_runtime_parameter_injection_prelude_materializes_consumer_global_eviden
+
+        script = (
+            "const hostConsole = console;\n"
+            "const logs = [];\n"
+            "globalThis.Memory = {};\n"
+            "globalThis.Game = {time: 88};\n"
+            "globalThis.console = {log: line => logs.push(String(line))};\n"
+            f"{uploaded_code}\n"
+            "module.exports.default.loop();\n"
+            "hostConsole.log(JSON.stringify({memory: globalThis.Memory.rlRuntimePolicyParameters, logs}));\n"
+        )
+        result = subprocess.run(
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)